requirements.txt

```diff
@@ -1,29 +1,42 @@
-# Used in local execution and dev.
-ipython
-requests
+##########################
+# Common (Local + Lambda)
+##########################
+#
+# These are the only dependencies which must get packaged
+# into every Lambda build.
 
-# Use PSL to determine base domains.
+strict-rfc3339
 publicsuffix
 
-# Used by all local and Lambda-based code.
-strict-rfc3339
 
-# To support a11y scanner:
-pyyaml
+#################
+# Common (Local)
+#################
+
+boto3
+ipython
 
-# Ubuntu 14.04 has an old version of six
-six>=1.6
 
-# To support pshtt scanner (and to stay fresh):
+############
+# Scanners
+############
+
+# pshtt
 git+git://github.com/dhs-ncats/pshtt.git#egg=pshtt
 
-# To support sslyze scanner:
+# sslyze
 sslyze
 cryptography
 
-# To support censys gatherer:
+# a11y
+pyyaml
+requests
+
+
+############
+# Gatherers
+############
+
+# censys
 google-cloud-bigquery
 google-auth-oauthlib
-
-# For Lambda support (only used locally):
-boto3
```

Comment:
I rearranged `requirements.txt` to much more clearly delineate which deps are used for which sections of the tool, to aid in creating tailored Lambda packages.

Reply:
I also removed the `requests` dependency, shrinking our dependency surface area.